src/Components/ContrastChecker/ContrastChecker.jsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from "react";
+import "./ContrastChecker.css";
+
+export default function ContrastChecker({ firstColor, secondColor }) {
+  const [contrastResult, setContrastResult] = useState(null);
+
+  useEffect(
+    (contrastResult) => {
+      async function fetchContrast() {
+        try {
+          const response = await fetch(
+            "https://www.aremycolorsaccessible.com/api/are-they",
+            {
+              method: "POST",
+              mode: "cors",
+              body: JSON.stringify({ colors: [firstColor, secondColor] }),
+              headers: {
+                "Content-Type": "application/json",
+              },
+            }
+          );
+
+          if (!response.ok) {
+            throw new Error(
+              `Failed to fetch data! Status Code: ${response.status}`
+            );
+          }
+
+          const contrastData = await response.json();
+          setContrastResult(contrastData.overall);
+
+          console.log("contrastData", contrastData);
+          console.log("contrastResult", contrastResult);
+        } catch (error) {
+          alert(`An error occurred: ${error.message}`);
```

Comment:
I tried hier many times but the alert keeps appearing and requires multiple clicks to dismiss..I think it has to do  using parameter in useEffect..It seems the fetch operation is triggered multiple times.

Reply:
I just removed the parameter and pushed once more.